TLM/TLM/State/OptionsTabs/PoliciesTab_InEmergenciesGroup.cs

```diff
@@ -0,0 +1,32 @@
+namespace TrafficManager.State {
+    using ICities;
+    using TrafficManager.UI;
+    using TrafficManager.UI.Helpers;
+
+    public static class PoliciesTab_InEmergenciesGroup {
+
+        public static CheckboxOption EvacBussesMayIgnoreRules =
+            new (nameof(Options.evacBussesMayIgnoreRules), Options.PersistTo.Savegame) {
+                Label = "VR.Checkbox:Evacuation buses may ignore traffic rules",
+                Validator = NaturalDisastersDlcValidator,
+            };
+
+        private static bool HasNaturalDisastersDLC
+            => SteamHelper.IsDLCOwned(SteamHelper.DLC.NaturalDisastersDLC);
+
+        internal static void AddUI(UIHelperBase tab) {
+            if (!HasNaturalDisastersDLC) return;
+
+            var group = tab.AddGroup(T("VR.Group:In case of emergency/disaster"));
+
+            EvacBussesMayIgnoreRules.AddUI(group);
+        }
+
+        private static bool NaturalDisastersDlcValidator(bool desired, out bool result) {
```

Comment:
Why this exists? Confusing function: updates some `out` variable and returns true? Move the logic to the caller.

Reply:
It's just a guard in central location at the moment. I'll probably change that in 11.6.5.2 branch but depends on how things go with converting `Options` to instance-based.